src/translations/en.json

```diff
@@ -5097,6 +5097,9 @@
               "select_users": "Select which users should see this view in the navigation"
             },
             "type": "View type",
+            "type_helper_sections": "You can not convert your view to 'sections' view type because migration is not supported yet. Start from scratch with a new view if you want to experiment 'sections' view.",
```

Comment:
```suggestion
            "type_helper_sections": "You can not change your view to use the 'sections' view type, because migration is not supported yet. Start from scratch with a new view if you want to experiment with the 'sections' view.",
```